DejaVu Sans is proportional font and looks pretty bad in terminal
window. Use GNU Unifont instead.
Index: grub-2.02~beta2/themes/starfield/theme.txt
===================================================================
--- grub-2.02~beta2.orig/themes/starfield/theme.txt
+++ grub-2.02~beta2/themes/starfield/theme.txt
@@ -25,7 +25,7 @@ message-font: "DejaVu Sans Regular 12"
 message-color: "#000"
 message-bg-color: "#fff"
 terminal-box: "terminal_box_*.png"
-terminal-font: "DejaVu Sans Regular 12"
+terminal-font: "Gnu Unifont Mono Regular 16"
 desktop-image: "starfield.png"
 
 #help bar at the bottom
